wp-content/themes/core/components/content_block/Controller.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace Tribe\Project\Templates\Components\content_block;
+
+use Tribe\Libs\Utils\Markup_Utils;
+use Tribe\Project\Templates\Components\Abstract_Controller;
+use Tribe\Project\Templates\Components\Deferred_Component;
+
+/**
+ * Class Content_Block
+ */
+class Controller extends Abstract_Controller {
+	/**
+	 * @var string
+	 */
+	public $tag;
+	/**
+	 * @var string[]
+	 */
+	private $classes;
+	/**
+	 * @var string[]
+	 */
+	private $attrs;
+	/**
+	 * @var string
+	 */
+	private $layout;
+	/**
+	 * @var Deferred_Component
+	 * @uses components/text
+	 */
+	private $leadin;
+	/**
+	 * @var Deferred_Component
+	 * @uses components/text
+	 */
+	private $title;
+	/**
+	 * @var Deferred_Component
+	 * @uses components/container
+	 */
+	private $content;
+	/**
+	 * @var Deferred_Component
+	 * @uses components/link
+	 */
+	private $cta;
+
+	public const LAYOUT_LEFT    = 'left';
+	public const LAYOUT_CENTER  = 'center';
+	public const LAYOUT_STACKED = 'stacked';
+	public const LAYOUT_INLINE  = 'inline';
+
+	public function __construct( array $args = [] ) {
+		$args = wp_parse_args( $args, $this->defaults() );
+
+		foreach ( $this->required() as $key => $value ) {
+			$args[ $key ] = array_merge( $args[ $key ], $value );
+		}
+
+		$this->tag     = $args['tag'];
+		$this->classes = (array) $args['classes'];
+		$this->attrs   = (array) $args['attrs'];
+		$this->layout  = $args['layout'];
+		$this->leadin  = $args['leadin'];
+		$this->title   = $args['title'];
+		$this->content = $args['content'];
+		$this->cta     = $args['cta'];
+	}
+
+	protected function defaults(): array {
+		return [
+			'tag'     => '',
+			'classes' => [],
+			'attrs'   => [],
+			'layout'  => self::LAYOUT_LEFT,
+			'leadin'  => null,
+			'title'   => null,
+			'content' => null,
+			'cta'     => null,
+		];
+	}
+
+	protected function required(): array {
+		return [
+			'tag'     => 'div',
```

Comment:
same comment here. If div is required, let's make it static and remove the option.

Reply:
Hah, that's just my bad, good catch. Set it to be the default vs. required.